pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Doesn't looks like pylint issues fix.

Reply:
This commit also contains fixes for tests